jaxb-ri/runtime/impl/pom.xml

```diff
@@ -28,6 +28,7 @@
     <packaging>jar</packaging>
     <name>JAXB Runtime</name>
     <description>JAXB (JSR 222) Reference Implementation</description>
+    <url>https://eclipse-ee4j.github.io/jaxb-ri/</url>
```

Comment:
there are probably more places where this needs to be applied. Ie is it correct in `jaxb-ri/bundles/osgi/osgi/pom.xml`?

Also note the while it should not be mandatory to sign-off commits any more, ECA needs to be signed. Right now, ECA check is failing and I see no way to verify that ECA has been signed